src/explore-education-statistics-admin/src/pages/release/datablocks/components/FeaturedTablesTable.tsx

```diff
@@ -0,0 +1,197 @@
+                  (featuredTable, index) => (
+                    <DraggableItem
+                      hideDragHandle
+                      id={featuredTable.id}
+                      index={index}
+                      isReordering={isReordering}
+                      key={featuredTable.id}
+                      tag="tr"
+                    >
+                      <FeaturedTablesRow
+                        canUpdateRelease={canUpdateRelease}
+                        dataBlock={dataBlocks.find(
+                          dataBlock =>
+                            dataBlock.id === featuredTable.dataBlockId,
+                        )}
+                        featuredTable={featuredTable}
+                        isReordering={isReordering}
+                        linkPath={generatePath<ReleaseDataBlockRouteParams>(
+                          releaseDataBlockEditRoute.path,
+                          {
+                            publicationId,
+                            releaseId,
+                            dataBlockId: featuredTable.dataBlockId,
+                          },
+                        )}
+                        onDelete={onDelete}
+                      />
+                    </DraggableItem>
+                  ),
+                )}
+              </DroppableArea>
+            </table>
+          )}
+        </Droppable>
+      </DragDropContext>
+    </>
+  );
+}
+
+interface FeaturedTablesRowProps {
+  canUpdateRelease: boolean;
+  dataBlock?: ReleaseDataBlockSummary;
+  featuredTable: FeaturedTable;
+  isReordering: boolean;
+  linkPath: string;
+  onDelete: (dataBlock: ReleaseDataBlockSummary) => void;
+}
+
+function FeaturedTablesRow({
+  canUpdateRelease,
+  dataBlock,
+  featuredTable,
+  isReordering,
+  linkPath,
+  onDelete,
+}: FeaturedTablesRowProps) {
+  if (!dataBlock) {
+    return null;
+  }
+  return (
+    <>
+      <td>{dataBlock.name}</td>
+      <td>{dataBlock.chartsCount > 0 ? 'Yes' : 'No'}</td>
+      <td>{dataBlock.inContent ? 'Yes' : 'No'}</td>
+      <td>{featuredTable.name}</td>
+      <td>
+        {dataBlock.created ? (
+          <FormattedDate format="d MMMM yyyy HH:mm">
+            {dataBlock.created}
+          </FormattedDate>
+        ) : (
+          'Not available'
+        )}
+      </td>
+      <td className="govuk-table__cell--actions govuk-!-width-one-quarter">
+        {!isReordering ? (
+          <>
+            <Link className="govuk-!-margin-bottom-0" unvisited to={linkPath}>
+              {canUpdateRelease ? 'Edit block' : 'View block'}
+            </Link>
+            {canUpdateRelease && (
+              <ButtonText
+                className="govuk-!-margin-bottom-0"
+                onClick={() => onDelete(dataBlock)}
+              >
+                Delete block
+              </ButtonText>
+            )}
+          </>
+        ) : (
+          <DragHandle className="dfe-position--relative govuk-!-margin-right-2" />
```

Comment:
Do we need the `dfe-position--relative`?

Reply:
Nope, it's gone now.